Document remote cluster setup within the same k8s cluster (#2718)
elastic/cloud-on-k8s: docs/orchestrating-elastic-stack-applications/elasticsearch/remote-clusters.asciidoc

[id="{p}-remote-clusters-connect-internal"]
== Connect from an Elasticsearch cluster running in the same Kubernetes cluster

> NOTE: The remote clusters feature requires a valid Enterprise license or Enterprise trial license. See <<{p}-licensing, the license documentation>> for details how to install your license.

Reviewer: ```suggestion
NOTE: The remote clusters feature requires a valid Enterprise license or Enterprise trial license. See <<{p}-licensing, the license documentation>> for more information about managing licenses.
```

> You can create a remote cluster connection to another Elasticsearch cluster inside the same Kubernetes cluster by specifying the `remoteClusters` attribute in your Elasticsearch spec.

Reviewer: ```suggestion
You can create a remote cluster connection to another Elasticsearch cluster deployed within the same Kubernetes cluster by specifying the `remoteClusters` attribute in your Elasticsearch spec.
```
